ui-core/src/main/java/com/adyen/checkout/components/ui/AddressDelegate.kt

```diff
@@ -6,7 +6,7 @@
  * Created by onurk on 3/1/2023.
  */
 
-package com.adyen.checkout.card
+package com.adyen.checkout.components.ui
```

Comment:
this should be inside `components-core`

Reply:
I am not sure that is the correct place. ButtonDelegate is in ui-core so AddressDelegate should be in the same place. If I move it, I have to move UI-related classes such as AddressOutputData, AddressInputModel

Reply:
In my opinion `ButtonDelegate` is specific for a UI element which is a `Button`, but  for `AddressDelegate` it has output and input more for a complete layout not just for a single view or something also we can discuss this with rest of the team.

Reply:
I discussed with Joseph and Ozgur.  we decided that it should be in ui-core